client/cl_utils.lua

```diff
@@ -7,3 +7,14 @@ function comma_value(amount)
 
     return amount
 end
+
+---@return table
+function FindPlayers()
+	local playerCoords = GetEntityCoords(cache.ped)
+    local closePlayers = {}
+    for _, v in pairs(GetPlayersFromCoords(playerCoords, 10)) do
+        closePlayers[#closePlayers+1] = GetPlayerServerId(v)
```

Comment:
v should already be the serverId, is it not?

Reply:
No when I was testing I would get 127,128,129 etc. when my server id was 1,2,3 etc.

Reply:
Disregard. I see you tested it.